app/assets/stylesheets/psique/components/_fonticon.scss

```diff
@@ -0,0 +1,113 @@
+// *************************************
+//
+//   Custom Font
+//   -> Font implementation
+//
+// *************************************
+
+// -------------------------------------
+//   Font-embedded
+// -------------------------------------
+
+@font-face {
+  font-family: 'rv-iconic';
+  src: asset-url('rv-iconic.eot?5591529');
+  src: asset-url('rv-iconic.eot?5591529#iefix') format('embedded-opentype'),
+       asset-url('rv-iconic.woff?5591529') format('woff'),
+       asset-url('rv-iconic.ttf?5591529') format('truetype'),
+       asset-url('rv-iconic.svg?5591529#rv-iconic') format('svg');
+  font-weight: normal;
+  font-style: normal;
+}
+/* Chrome hack: SVG is rendered more smooth in Windozze. 100% magic, uncomment if you need it. */
+/* Note, that will break hinting! In other OS-es font will be not as sharp as it could be */
+/*
+@media screen and (-webkit-min-device-pixel-ratio:0) {
+  @font-face {
+    font-family: 'rv-iconic';
+    src: url('../font/rv-iconic.svg?5591529#rv-iconic') format('svg');
+  }
+}
+*/
+ 
+// -------------------------------------
+//   Font-setup
+// -------------------------------------
+
+[class^="icon-"]:before, [class*=" icon-"]:before {
```

Comment:
Why not instead of that `[class^="icon-"]:before, [class*=" icon-"]:before` we use the `icon` class for common styles?
I've seen bootstrap & foundation are doing like yours, I don't get it =/
Anyone can explain it to me?